.github/workflows/ci.yml

```diff
@@ -151,7 +151,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        repoVersion: [ 7.3.2, 7.4.2, 23.3.0 ]
+        repoVersion: [ 7.3.2, 7.4.2, 23.3.0, 23.4.0-M1 ]
```

Comment:
I don't think we should test both. We're only testing one patch for each LTS version 7.3 and 7.4, and so we should only test one patch for the LTS version 23 (nb. the LTS strategy changed between 7.4.x and 23.x).

Reply:
I've rollback the changes for ci.yml file so it keeps 23.3.0